src/plugins/cordova-plugin-battery-status/battery.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var db = require('db'),
+    constants = require('sim-constants');
+
+var _callbacks = {
+        sendPluginResult: null,
+        error: null
+    },
+    _batteryInfo = {};
+
+var battery = {
+    listenerEnabled: false,
+
+    initialize: function (message) {
+        var isPlugged = db.retrieve(constants.BATTERY_STATUS.IS_PLUGGED_KEY);
+        _batteryInfo.level = db.retrieve(constants.BATTERY_STATUS.BATTERY_STATUS_KEY) || 100;
+        _batteryInfo.isPlugged =  (isPlugged) ? (isPlugged === 'true') : true;
```

Comment:
@TimBarham  I found an issue here that I've fixed. Boolean values are retrieved as string from db, so I've modified this in order to save the boolean value of isPlugged, otherwise the fallback is true.
When the saved value was false, in the previous implementation, the isPlugged value ends up being true, that was wrong.